Casamari/MNC001.xml

```diff
@@ -341,45 +740,160 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                          The note is written in Amharic in the upper margin of the folio and is encircled in two rectangular frames.
+                          It contains the names of the leaders who divide the four parts of the year and of those who lead them, as narrated in
+                           <ref type="work" corresp="LIT1340EnochE.3.82"/>. 
+                        </desc>
+                        <locus target="#69v"/>
+                        <q xml:lang="am">
+                           <seg part="I">
+                              ምልክኤል፡ ሕልምልሜሌክ፡ ምልኤል፡ ናርርል። የዘጠና። አድናርኤል፡ ኢየሱሳኤል፡ ኢየሉምኤል፡ የወ። መ። 
+                              ለና፡ ብርክኤል፡ ዜልብስኤል፡
+                           </seg>
+                           <seg part="F">
+                              አድናርኤል፡ ኢየሱሳኤል፡ ኢየሉምኤል፡ አስፋ፡ ፬የወ፡ መ፡ ለ፬፡ ብርክኤል፡ ዜልብስኤል፡ ሄሉያሴፍ፡ የለ፡ የሻ፡ 
+                              ለምል፡ ቤዳ፡ ኬኤ፡ ሄኬ፡ የለ፡ የሻ፡ ለህ 
+                           </seg>
+                        </q>
+                     </item>
+                     
+                     <item xml:id="a6">
+                        <desc type="Comment">         
+                           The note is written in the upper margin of the folio, in corrispondence of <ref type="work" corresp="LIT1688Job.36"/>.
+                        </desc>
+                        <locus target="#93vb"/>
+                        <q xml:lang="gez">
+                           በሐልዮ፡ በነቢብ፡ በገቢር፡ በሕዛንነት፡ በቁልማስነት፡ በሽምግልና፡ በጥዒና። በደዌ፡ በሞት፡ በደዌ፡ በሞት፡
+                           በመቃብር፡ በሞት፡ በመቃብር፡ በትንሣኤ፡ በንጽሐ፡ ሥጋ፡ በንጽሐ፡ ልብ፡ በንጽሐ፡ ነፍስ፡ የባላ፡ 
+                           ፴ የባላ<supplied reason="omitted" resp="MV">፡</supplied> ፰፡ የባላ<supplied reason="omitted" resp="MV">፡</supplied> ፻፡
+                        </q>
+                     </item>
+                     
+                     <item xml:id="a7">
+                        <desc type="OwnershipNote">       
+                          The note is crudely written in black ink. It mentions <persName ref="PRS12015TaklaSe" role="owner"><roleName type="title">ʾalaqā</roleName> Takla Ṣǝyon</persName> 
+                           as the owner of the manuscript.
+                        </desc>
+                        <locus target="#97vb"/>
+                        <q xml:lang="gez">
+                           ዝመጽሐፍ፡ ዘአለቃ፡ ተክለ<supplied reason="omitted" resp="MV">፡</supplied> ጽዮን፡ ዘሰረቆ፡ ወዘፈሐቆ፡ ለይኩን፡ ለይኩን፡
+                        </q>
+                     </item>
+                    
+                        <item xml:id="e1">
+                           <desc>A sticker of the monastic library with the signature ‘Ms 29’ is glued onto the inner side of the front board. 
+                              The inventory number "50426" is written in pencil in the lower margin of <locus target="#156v"/>.</desc>
                         </item>
-                        </list>
-                     -->
+                        <item xml:id="e2">
```

Comment:
It would be nice to look for a nice example or even two to be printed in our catalogue.
I started to collect images from the mss I encode. Of course, later we decide all together. However, it might be useful to look for additional material already now.